DFC.Digital/DFC.Digital.AcceptanceTest/Infrastructure/Pages/ContactAnAdviserFormsPage.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TestStack.Seleno.PageObjects.Locators;
+
+namespace DFC.Digital.AcceptanceTest.Infrastructure.Pages
+{
+    public class ContactAnAdviserFormsPage : DFCPage
+    {
+        public bool ContactQuestionDisplayed => Find.Element(By.Id("ContactAdviserQuestionType")) != null;
+
+        public bool UserDetailsFormDisplayed => Find.Element(By.Id("userform")) != null;
+
+        public bool ErrorValidationDisplayed => Find.Element(By.Id("error-validation-summary")) != null;
+
+        public string QuestionTypeErrorMessage => Find.Element(By.Id("ContactAdviserQuestionType-error")).Text;
+
+        public string AdviserQueryErrorMessage => Find.Element(By.Id("Message-error")).Text;
+
+        public string DateOfBirthErrorMessage => Find.Element(By.Id("DateOfBirth-error")).Text;
+
+        public void CompleteContactAdviserInitialForm(string option, string query)
+        {
+            SelectContactOption(option);
+            EnterText("Message", query); //Enters text into field ID 'message' which is the adviser field on the First contact adviser form
+        }
+
+        public void CompleteSecondForm(string firstName, string lastName, string email, string confEmail, string dob, string postcode)
+        {
+            EnterText("Firstname", firstName);
```

Comment:
Why are we not using ViewModel for data entry?

Reply:
As per conversation, added comment above the method to advise why this is now being used